pkg/commands/compute/init_test.go

```diff
@@ -21,6 +22,10 @@ func TestInit(t *testing.T) {
 		t.Skip("Set TEST_COMPUTE_INIT to run this test")
 	}
 
+	// TODO: Change back to "main" before merging PR.
```

Comment:
@triblondon in order for the tests to show as passing I needed to use a PR branch for each of the starter kits we pull in. 

So basically once this PR is approved, and all the starter kit PRs are approved (see: links below), then I'll need to coordinate merging both this PR and cut a new CLI release as quickly as possible after the fact.

- https://github.com/fastly/compute-starter-kit-rust-default/pull/49
- https://github.com/fastly/compute-starter-kit-assemblyscript-default/pull/21
- https://github.com/fastly/compute-starter-kit-javascript-default/pull/7
- https://github.com/fastly/compute-starter-kit-rust-beacon-termination/pull/16
- https://github.com/fastly/compute-starter-kit-rust-static-content/pull/13

> NOTE: with the exception of the rust starter kit they're all one line changes, so if you could also take a moment to approve those (when you have a chance).